dist/tools/doccheck/check.sh

```diff
@@ -6,17 +6,16 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-RIOTBASE=$(readlink -f "$(dirname $(realpath $0))/../../..")
+RIOTBASE="$(cd $(dirname $0)/../../..; pwd)"
 
 ERRORS=$(make -C "${RIOTBASE}" doc 2>&1 | \
             grep '.*warning' | \
             sed "s#${PWD}/\([^:]*\)#\1#g")
 
-if [ -n "${ERRORS}" ]
-then
+[ -n "${ERRORS}" ] && {
```

Comment:
I personally prefer the `if` version.

Reply:
mhm, I found many of this (non-explicit-if) variants in our scripts ...

But we agree that the `else` part is not needed because of the `exit 2` right?

Reply:
Yes

Reply:
this rendered obsolete with recent changes to the script by other PR(s)